### DISCUSSION TITLE
Menu block setting for specifying flex direction

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1943] [Ideas]
It would be very helpful to add a setting to the Menu block's navbar-nav for setting the flex-direction to column or row. The only way to do it now is to manually add a style.

**Comment by meetmandeep** [comment]
@eswanzey Vanjaro is user/feature focused rather than developer focused. Can you please include a use case? What exactly are you trying to accomplish?
